registration/registration-services/src/main/java/io/mosip/registration/constants/AuditEvent.java

```diff
@@ -311,7 +311,9 @@ public enum AuditEvent {
 	REG_BLOCKLISTED_WORD_ACCEPTED("REG-EVT-102", USER_EVENT.getCode(), "REG_BLOCKLISTED_WORD_ACCEPTED",
 			"Demographic Details : A blocklisted word has been accepted in Registration"),
 	REG_BLOCKLISTED_WORD_REJECTED("REG-EVT-103", USER_EVENT.getCode(), "REG_BLOCKLISTED_WORD_REJECTED",
-			"Demographic Details : A blocklisted word has been rejected in Registration");
+			"Demographic Details : A blocklisted word has been rejected in Registration"),
+	REG_BIO_CAPTURE_DETAILS("REG-EVT-104", USER_EVENT.getCode(), "REG_BIO_CAPTURE_DETAILS",

```

Comment:
we will also need the modality name that was captured. Also do we have an audit for failed captures ?

Reply:
added modality name and failure case also...